src/libs/print_settings.c

```diff
@@ -997,6 +1006,19 @@ gui_init (dt_lib_module_t *self)
 
   g_signal_connect (G_OBJECT (d->pintent), "value-changed", G_CALLBACK (_printer_intent_callback), (gpointer)self);
 
+  d->bpc = gtk_check_button_new_with_label(_("black point compensation"));
+  gtk_box_pack_start(GTK_BOX(self->widget), GTK_WIDGET(d->bpc), TRUE, FALSE, 0);
+  g_signal_connect(d->bpc, "toggled", G_CALLBACK(_printer_bpc_callback), (gpointer)self);
+
+  if (!dt_conf_key_exists("plugins/print/print/black_point_compensation"))
```

Comment:
Adding it with the default `TRUE` to data/darktableconfig.xml.in would be better.